data/init/dfhack.keybindings.init

```diff
@@ -25,6 +25,9 @@ keybinding add Ctrl-Shift-K gui/cp437-table
 # customizable quick command list
 keybinding add Ctrl-Shift-A gui/quickcmd
 
+# save the description of a selected unit or item to the `/markdown_{YourWorldName}.md` file in the root game directory
+keybinding add Shift-R@dwarfmode/ViewSheets/UNIT|dwarfmode/ViewSheet markdown
```

Comment:
I'm vary wary about adding a binding to regular text keys. It can cause non-obvious problems with text entry. For example, it would prevent a squad from having a name with a capital R in it:
![image](https://github.com/DFHack/dfhack/assets/977482/b4c22f25-47c1-4f0c-9983-0ffd92d110f5)

I suggest Alt-R instead

```
dwarfmode/ViewSheet
```
I think this got truncated. This is supposed to be `dwarfmode/ViewSheets/ITEM`, right?

Also, please move this down to the dwarfmode section below.